lib/lecture.rb

```diff
@@ -56,7 +61,7 @@ def download_video
         "wget",
         video_url,
         "-c",
-        "-O",  "#{name}.mp4",
+        "-O",  "#{name}.avi",
```

Comment:
Any reason this is changed to .avi instead of mp4? Is this course specific video format being provided by the course you're downloading?